storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -29,6 +29,44 @@ In future {product-title} versions, volumes provisioned using existing in-tree p
 After full migration, in-tree plug-ins will eventually be removed in future versions of {product-title}.
 ====
 
+=== Storage policy
+
+The vSphere CSI Operator Driver storage class now uses vSphere's storage policy. {product-title} automatically creates a storage policy that targets datastore configured in cloud configuration:
+[output, yaml]
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: thin-csi
+provisioner: csi.vsphere.vmware.com
+parameters:
+  StoragePolicyName: "${STORAGE_POLICY_NAME}"
+volumeBindingMode: WaitForFirstConsumer
+# Volume expansion is disabled here because volume expansion in the vSphere
+# CSI driver is still in beta and has several limitations. (see
+# https://vsphere-csi-driver.sigs.k8s.io/supported_features_matrix.html).
+# It is not supported for all datastores (vSAN only right now) and there is
+# not yet an e2e test for it.
+allowVolumeExpansion: false
+reclaimPolicy: Delete
+----
+
+=== Version requirements
+
+The vSphere CSI Operator Driver requires certain minimum versions for products that make up the virtual environment as shown in the following table.
```

Comment:
```suggestion
The vSphere CSI Operator Driver requires certain minimum versions for products that make up the virtual environment as shown in the following table:
```

Reply:
I could not find a reference to how to introduce a table in RH or IBM's SGs. I recall that Chicago and I believe MS recommend a sentence ending with a period, rather than a colon.